contributing.md

```diff
@@ -289,6 +289,66 @@ ansible-playbook --skip-tags tests::cleanup \
     -i testhost, tests/playbooks/tests_802_1x.yml
 ```
 
+### Integration tests with podman
+1. Create `~/.ansible/collections/ansible_collections/containers/podman/` if this
+directory does not exist and `cd` into this directory.
+```bash
+mkdir -p ~/.ansible/collections/ansible_collections/containers/podman/
+cd ~/.ansible/collections/ansible_collections/containers/podman/
+```
+
+2. Clone the collection plugins for Ansible-Podman into the current directory.
+```bash
+git clone https://github.com/containers/ansible-podman-collections.git .
+```
+
+3. Change directory into the `network/tests`, `network` is the Repo you cloned from
+your local fork.
+```bash
+cd ~/network/tests
+```
+
+
+4. Use podman with `-d` to run in the background (daemon). Use `c7` because
+`centos/systemd` is centos7.
+```bash
+podman run --name c7 --rm --privileged -v /sys/fs/cgroup:/sys/fs/cgroup:ro -d registry.centos.org/centos/systemd
+```
+
+5. Use `podman unshare` first to run "podman mount" in root mode, use `-vi` to run
+ansible as inventory in verbose mode, use `-c podman` to use collection plugins. Note,
+the failed test files are:
+- `tests_802_1x_nm.yml` 
+- `tests_802_1x_updated_nm.yml`
+- `tests_bond_initscripts.yml`
+- `tests_bond_nm.yml`
+- `tests_bridge_initscripts.yml`
+- `tests_bridge_nm.yml`
+- `tests_default_nm.yml`
```

Comment:
What's the failure for this one?

Reply:
- `tests_bond_nm.yml`

TASK [linux-system-roles.network : Enable and start NetworkManager] **************************************************************************************************************************************************************************
fatal: [c7]: FAILED! => {"censored": "the output has been hidden due to the fact that 'no_log: true' was specified for this result", "changed": false}

TASK [linux-system-roles.network : Enable and start NetworkManager] **************************************************************************************************************************************************************************
fatal: [c7]: FAILED! => {"censored": "the output has been hidden due to the fact that 'no_log: true' was specified for this result", "changed": false}
...ignoring

TASK [linux-system-roles.network : Configure networking connection profiles] *****************************************************************************************************************************************************************
fatal: [c7]: FAILED! => {"_invocation": {"module_args": {"__debug_flags": "", "connections": [{"name": "bond0.1", "persistent_state": "absent", "state": "down"}, {"name": "bond0.0", "persistent_state": "absent", "state": "down"}, {"name": "bond0", "persistent_state": "absent", "state": "down"}], "force_state_change": false, "ignore_errors": false, "provider": "nm"}}, "changed": false, "msg": "fatal error: failure loading libnm library: g-io-error-quark: Could not connect: No such file or directory (1)", "stderr": "\n", "stderr_lines": [""]}
...ignoring

TASK [command] *******************************************************************************************************************************************************************************************************************************
fatal: [c7]: FAILED! => {"changed": true, "cmd": ["ip", "link", "del", "nm-bond"], "delta": "0:00:00.032600", "end": "2020-09-11 01:43:47.010923", "msg": "non-zero return code", "rc": 1, "start": "2020-09-11 01:43:46.978323", "stderr": "Cannot find device \"nm-bond\"", "stderr_lines": ["Cannot find device \"nm-bond\""], "stdout": "", "stdout_lines": []}
...ignoring


PLAY RECAP ***********************************************************************************************************************************************************************************************************************************
c7                         : ok=24   changed=4    unreachable=0    failed=1    skipped=7    rescued=0    ignored=3   








- `tests_bridge_initscripts.yml`

TASK [assert that interface LSR-TST-br31 is absent] ******************************************************************************************************************************************************************************************
fatal: [c7]: FAILED! => {
    "assertion": "not interface_stat.stat.exists",
    "changed": false,
    "evaluated_to": false,
    "msg": "LSR-TST-br31 exists"
}

PLAY RECAP ***********************************************************************************************************************************************************************************************************************************
c7                         : ok=12   changed=1    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   








- `tests_bridge_nm.yml`

TASK [assert that interface LSR-TST-br31 is absent] ******************************************************************************************************************************************************************************************
fatal: [c7]: FAILED! => {
    "assertion": "not interface_stat.stat.exists",
    "changed": false,
    "evaluated_to": false,
    "msg": "LSR-TST-br31 exists"
}

PLAY RECAP ***********************************************************************************************************************************************************************************************************************************
c7                         : ok=12   changed=1    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   









- `tests_default_nm.yml`

TASK [linux-system-roles.network : Enable and start NetworkManager] **************************************************************************************************************************************************************************
fatal: [c7]: FAILED! => {"censored": "the output has been hidden due to the fact that 'no_log: true' was specified for this result", "changed": false}

PLAY RECAP ***********************************************************************************************************************************************************************************************************************************
c7                         : ok=6    changed=0    unreachable=0    failed=1    skipped=2    rescued=0    ignored=0